package.json

```diff
@@ -35,7 +35,7 @@
     "eslint-plugin-import": "^2.14.0",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.12.2",
-    "eslint-plugin-react-hooks": "^1.0.0"
+    "eslint-plugin-react-hooks": "^2.0.1"
```

Comment:
Are we major version bumping this package again?

Reply:
[My understanding was this version adds a lint rule around usage of hooks at top level which crashes the app](https://github.com/facebook/react/pull/16455). 

So before if you used hooks at top level; the app will crash. With pulling in this version now, if you use hooks at top level, you'll get a lint warning + app still crashes.

Based on that I'm leaning towards treating this as a minor release, however the react team [considered this a breaking change](https://github.com/facebook/react/pull/16455#issuecomment-522707242).

Reply:
my vote is ship it as a patch release lol